### PR TITLE
JBIDE-21327 - most recent openshift-vagrant requires tls-verify flag …

### PR DESCRIPTION
**Pull request by robstryker**
…to be set properly

### CODE REVIEW

File: plugins/org.jboss.tools.openshift.cdk.server/src/org/jboss/tools/openshift/cdk/server/core/internal/listeners/ConfigureDependentFrameworksListener.java [hunk 1]
```diff
@@ -109,12 +109,10 @@ private void configureDocker(IServer server, ADBInfo adb) {
 						.name(getNextName(server, mgr)).tcpHost(dockerHost);
 				String tlsVerifyString = adb.env.get("DOCKER_TLS_VERIFY");
 				boolean tlsVerify = (Integer.parseInt(tlsVerifyString) != 0);
```
Comment (maxandersen) [inline plugins/org.jboss.tools.openshift.cdk.server/src/org/jboss/tools/openshift/cdk/server/core/internal/listeners/ConfigureDependentFrameworksListener.java]: afaik just the presence of DOCKER_TLS_VERIFY is enough - the value does not matter.

**Comment by maxandersen** [comment]
looks safe and valid to apply

**Comment by maxandersen** [comment]
+1 for branch and master.

**Comment by robstryker** [comment]
pushed after rebase

**Pull request closed without merging**